packages/console-types/src/types/endpoints.ts

```diff
@@ -47,6 +47,32 @@ const routeBooleanValue = {
   },
 } as const
 
+const routeRequestsPerSecond = {
+  type: 'object',
+  default: { inherited: true },
+  if: { type: 'object', properties: { inherited: { type: 'boolean', const: false } } },
+  then: {
+    type: 'object',
+    properties: {
```

Comment:
Almost. It has to include also the `inherited` property.

Reply:
I know, I worry about the other properties, wondering if we could merge the by spreading the shared properties, but i guess this only applies to requestsPerSecond so I think this can be fine